### PR TITLE
feat: added images and inputs components, variables updation

### PR DESCRIPTION
**Pull request by rishbitsnbytes**
Please review this component : 

Added Images and Inputs Components.

- Images with responsive and round component.
- Inputs Default and Validation of Warning, Success and Error Feedback.
- Input Textbox.

Preview :

images

inputs

### CODE REVIEW

File: Components/Images-Inputs/images-inputs.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+.img-round {
+    width: 10rem;
+    height: 10rem;
+    border-radius: 50%;
+}
+
+
+/* <!-- Inputs default --> */
+
+.input-grp {
```
Comment (bharati-21) [inline Components/Images-Inputs/images-inputs.css]: Suggestion to separate styles for components.

File: Components/Images-Inputs/images-inputs.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    <img class="img-responsive" src="/Assets/cosmos8.jpg" alt="Image">
+
+    <img class="img-responsive img-round" src="/Assets/cosmos8.jpg" alt="Image">
+
+
+
+    <!-- Input default -->
+
+    <div class="input-grp">
```
Comment (bharati-21) [inline Components/Images-Inputs/images-inputs.html]: The same suggestion here as well.

**Review by bharati-21 — APPROVED** [review]
Looks good to be merged!

**Review by NeerajVodala — APPROVED** [review]
Separate stylesheet for each component is recommended(as Bharati said).
Rest everything looks fine to me.
All the best.

File: Components/Images-Inputs/images-inputs.css (hunk 1 above)
Comment (rishbitsnbytes) [reply Components/Images-Inputs/images-inputs.css]: Okay thanks for the suggestion. Refactored it.

**Comment by rishbitsnbytes** [comment]
> Separate stylesheet for each component is recommended(as Bharati said). Rest everything looks fine to me. All the best.

Okay, thanks, It has been refactored.

**Pull request merged**